qemu/qemu_test.go

```diff
@@ -240,6 +239,7 @@ func TestAppendKnobsAllTrue(t *testing.T) {
 }
 
 func TestAppendKnobsAllFalse(t *testing.T) {
+	var knobsString = "-realtime mlock=off"
```

Comment:
so you cannot explicitly turn off mlock without the -realtime option if I am understanding my quick google of the option. It appears that -realitme on it's own does nothing, but if you don't explicitly do -mlock=off, the default is mlock=on, which you want to make configurable. Right?

Reply:
Agree. Today it does nothing else.. I will reword accordingly